docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,482 @@
+  master-eligible nodes in the cluster.
+
+* Make entirely sure the lost master-eligible nodes cannot be repaired or
+  recovered by moving their data paths to healthy hardware
+
+* Stop all remaining nodes
 
-   Current node cluster state (term, version) pair is (4, 12)
+* Choose one of the remaining master-eligible nodes to become the new elected
+  master, as described in :ref:`crate-node-cli-repurposing-a-node-as-master`
+
+* On the chosen node, run the ``unsafe-bootstrap`` command as demonstrated
+  below. Verify that the tool-reported master node was successfully
+  bootstrapped
+
+* Start this node and verify that it is the elected master node
+
+* Run the ``detach-cluster`` tool as demonstrated below on every other node in
+  the cluster
+
+* Start all other nodes and verify that each node joins the cluster
+
+* Investigate the data in the cluster to discover if any data was lost during
+  the process
+
+When you run the tool it makes sure that the node used to bootstrap the cluster
+is not running. It is important that all other master-eligible nodes are also
+stopped while this tool is running.
+
+The message *Master node was successfully bootstrapped* means that the tool
+successfully completed its job. It does not say that there has not been a data
+loss.
+
+.. NOTE::
 
-If you have a choice of nodes on which to run this tool then you should choose
-one with a term that is as large as possible. If there is more than one node
-with the same term, pick the one with the largest version. This information
-identifies the node with the freshest cluster state, which minimizes the quantity
-of data that might be lost. For example, if the first node reports (4, 12) and a
-second node reports (5, 3), then the second node is preferred since its term is larger.
-However if the second node reports (3, 17) then the first node is preferred since its
-term is larger. If the second node reports (4, 10) then it has the same term as the
-first node, but has a smaller version, so the first node is preferred.
+  The tool does not self-reliantly check for not-running master-eligible nodes.
+  Make sure that all other master-eligible nodes are not running.
 
 .. WARNING::
 
-  These steps can lead to arbitrary data loss since the target node may not
-  hold the latest cluster metadata, and this out-of-date metadata may make
-  it impossible to use some or all of the tables in the cluster.
+  The next step can lead to arbitrary data loss since the target node may not
+  hold the latest cluster metadata. Outdated metadata may make it impossible
+  to use some or all of the tables in the cluster.
 
-The sequence of operations for using this tool are as follows:
+To unsafely bootstrap the cluster, run:
 
-* Make sure you have really lost access to at least half of the master-eligible
-  nodes in the cluster, and they cannot be repaired or recovered by moving their
-  data paths to healthy hardware.
+``crate-node unsafe-bootstrap``
```

Comment:
this command, and the other commands like it that appear on a single line, should be formatted as code blocks

please also use the `console` language setting for all code blocks. you can set this once at the start of the file. you just need to update the `highlight:: psql` line